frontend/src/metabase/nav/components/AppBar/AppBarLarge.styled.tsx

```diff
@@ -44,6 +46,31 @@ export const AppBarLeftContainer = styled.div<AppBarLeftContainerProps>`
       opacity: ${props => (props.isNavBarEnabled ? 1 : 0)};
     }
   }
+
+  ${props =>
+    !props.isLogoVisible &&
```

Comment:
I put everything under the case `css` block to avoid having the below logic spread across different CSS selectors.